composer.json

```diff
@@ -19,7 +19,7 @@
     }
   ],
   "require": {
-    "php": ">=5.6",
```

Comment:
Thanks for the clarification.

Other dependencies need phpecc 0.4 (like spomky-labs/jose), so composer will shout at you nevertheless :/ I'm also not sure about requiring 0.3, as this is a crypto library and I very much prefer having it at the latest version for security reasons. What do you think?